CHANGELOG.md

```diff
@@ -34,6 +36,15 @@ Important changes:
   * When building with `CMake` option `WITH_ABI_VERSION_1=ON` (by default)
     the `ABI` is unchanged, and the fix is not available.
 
+* [API] Add a new AddLink() operation to Span
+  [#2380](https://github.com/open-telemetry/opentelemetry-cpp/pull/2380)
+  * New `API` Span::AddLink() adds a single link to a span.
+  * New `API` Span::AddLinks() adds multiple links to a span.
+  * Because this is an `ABI` breaking change, the fix is only available
+    with the `CMake` option `WITH_ABI_VERSION_2=ON`.
```

Comment:
Is `WITH_ABI_VERSION_2` experimental and will be marked as stable sometime later?

Reply:
`WITH_ABI_VERSION_2` is still experimental.

At some point we can make the decision to declare ABI v2 stable, and also to deprecate ABI v1, but this will be much later, when there are no more changes expected in ABI v2.